core/domain/param_domain.py

```diff
@@ -144,6 +144,11 @@ def customization_args(self):
         return self._customization_args
 
     def to_dict(self):
+        """Returns a dict representing this Param domain object.
```

Comment:
This is wrong. It's a ParamChange domain object. Ditto below.

Reply:
Sorry for that...